dist/main.js

```diff
@@ -0,0 +1,54 @@
+$('#submitBtn').on('click', () => {
+    const input = $('#teamNameInput').val()
+
+    $.get(`/teams/${input}`, function(result) {
+        renderer.render(result) 
+    })
+})
+
+$('#dreamTeamBtn').on('click', () => {
+    $.get('/dreamTeam', (result) => {
+        renderer.render(result)
+    })
+})
+
+$('#container').on('click','#add-btn', function() {
+    const playerContainer = $(this).closest('.player')
+    const name = playerContainer.find('#name').text().split(' ')
+    const player = {
+        firstName: name[0],
+        lastName: name[1],
+        jerseyNum: playerContainer.find('#jerseyNum').text(),
+        img: playerContainer.find('img').attr('src'),
+        position: playerContainer.find('#position').text(),
+        addBtn: 'Add to Dream Team',
+        delBtn: 'Delete from DreamTeam',
+        playerId: playerContainer.data().id
+    }
+    $.post(`/roster`, player)
```

Comment:
Add call back function to receive data from the server or catch the error if something happened with the request